packages/flutter_tools/test/integration.shard/test_utils.dart

```diff
@@ -4,36 +4,47 @@
 
 import 'dart:async';
 
-import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
 import 'package:flutter_tools/src/base/io.dart';
-import 'package:flutter_tools/src/globals.dart' as globals;
+import 'package:flutter_tools/src/base/platform.dart';
+import 'package:process/process.dart';
 
 import '../src/common.dart';
 
+/// The [FileSystem] for the integration test environment.
```

Comment:
How about a test in forbidden_imports_test.dart that checks that this file isn't imported outside of the integration.shard subdir?